protocol/src/test/scala/io/mediachain/protocol/DataObjectGenerators.scala

```diff
@@ -12,71 +13,134 @@ object DataObjectGenerators {
   import io.mediachain.util.cbor.CborAST._
   import io.mediachain.util.cbor.CValueGenerators._
 
+  val genStringMetas: List[Gen[(String, CValue)]] = (1 to 25).toList.map { _ =>
+    for {
```

Comment:
sure, feel free to make an issue with that specific gist, looks neat but prob not worth reworking right now